### PR TITLE
Chapter 12

### PR DESCRIPTION
**Pull request by bast**
3 smaller recipes coming up and then this chapter is done source code wise.

### CODE REVIEW

File: Chapter12/recipe-05/README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Using test labels
+# Dynamic analysis to detect memory defects
```
Comment (robertodr) [inline Chapter12/recipe-05/README.md]: :+1: great idea!

File: Chapter12/recipe-09/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Running a subset of tests
```
Comment (robertodr) [inline Chapter12/recipe-09/README.md]: Probably a better title would be "Setting test labels"?

Comment (bast) [reply Chapter12/recipe-09/README.md]: Possibly. The thing is that I also want to show how to run tests by their name (regex) and by their numbers so it is not only about labels. But if you find the title misleading, then I will change it.

Comment (robertodr) [reply Chapter12/recipe-09/README.md]: I had not thought about that at all! Keep this title, it's better.

Comment (bast) [reply Chapter12/recipe-09/README.md]: I'll go for "Setting test labels".

Comment (bast) [reply Chapter12/recipe-09/README.md]: OK :-) I'll keep it.

**Pull request merged**